damus/Views/AttachMediaUtility.swift

```diff
@@ -140,24 +140,24 @@ extension NSMutableData {
 
 enum ImageUploader: String, CaseIterable, Identifiable {
     var id: String { self.rawValue }
-    case nostrBuild
-    case nostrImg
+    case nostrbuild
+    case nostrimg
 
     var nameParam: String {
         switch self {
-        case .nostrBuild:
+        case .nostrbuild:
             return "\"fileToUpload\""
-        case .nostrImg:
+        case .nostrimg:
             return "\"image\""
         }
     }
 
     var displayImageUploaderName: String {
```

Comment:
It doesn't seem like you're using this variable. Remove?

Reply:
This computed var was used for generating the image uploader's name in the picker. Not sure how it's use case got removed